### PR TITLE
Lab 1 and 2 JS complete

### PR DESCRIPTION
**Pull request by jcortes89**
Lab 1 Javascript complete

### CODE REVIEW

**Comment by jcortes89** [comment]
Hey Bruce! just sent over the items I needed to fix in 11. Completed lab 9 as well and the HTML lab for today. Let me know if i need to do anything else!

**Review by brucestull — COMMENTED** [review]
[Lab 9: ATM](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/09%20ATM.md)
**Needs Work**
* Comments:
    * Interest on $100:
        * Actual interest accrued:
            * `100.1`
        * Expected interest accrued:
            * `.1`

[Lab 11: Quotes API](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/11%20Quotes%20API.md)
**Needs Work**
* Comments:
    * Need to remove pprint statements from console display for user. We want to use the keys to control what the user sees and not show them the whole dictionary response.

File: code/jose/Python_Labs/lab11.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    for i, value in enumerate(combined_quotes):
+        print(combined_quotes[i]['author'])
+        print(combined_quotes[i]['body'])
+
+    second_user_input = input('Would you like to view the next page or a new keyword?: ')
+
+    if second_user_input == 'new keyword':
+        keyword= input('What keyword would you like to search for?: ')
+        response = requests.get('https://favqs.com/api/quotes/?filter='+keyword+'&type=tag', headers = {'Authorization': 'Token token="855df50978dc9afd6bf86579913c9f8b"'})
```
Comment (brucestull) [inline code/jose/Python_Labs/lab11.py:55]: * Is there a reason you chose to use `&type=tag`? `filter` will automatically select quotes by `keyword`. Using `&type=tag` will severely reduce number of response quotes.

**Review by brucestull — CHANGES_REQUESTED** [review]
**Needs Work**

See https://github.com/PdxCodeGuild/class_swordfish/pull/117#pullrequestreview-1130309205

**Comment by jcortes89** [comment]
Lab 1 Javascript ready for review

**Comment by jcortes89** [comment]
Lab 1 and 2 are complete

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 1-3: Pick 3](https://github.com/PdxCodeGuild/class_swordfish/blob/main/4%20JavaScript/labs/Lab%2001-03%20Pick%203.md)

## Lab 01 - `lab01.js`
**Completed**


## Lab 02 - `lab02.js`
**Needs Work**
* Comments:
    * Many numbers under 10 do not give a response.
    * Input of `75` results in `fiftyseven`, when expected would be something like `seventy five`.

input
output


## Lab 03 - `THIRD_CHOICE`
**Not Submitted**

**Review by brucestull — APPROVED** [review]
Merging to start with clean pull request for future labs.

**Pull request merged**